.circleci/config.yml

```diff
@@ -188,23 +188,23 @@ jobs:
             echo "Your dependencies are secure enough: $SUMMARY"
             exit 0
 
-  #  testE2e:
-  #    executor: web
-  #    environment:
-  #      - BINARY_PATH: "/tmp/gaia/builds/Gaia/linux_amd64/gaiacli"
-  #      - NODE_BINARY_PATH: "/tmp/gaia/builds/Gaia/linux_amd64/gaiad"
-  #    steps:
-  #      - checkout
-  #      - attach_workspace:
-  #          at: *GAIA
-  #      - yarn-install
-  #      - run:
-  #          name: Test
-  #          command: yarn run test:e2e
-  #          no_output_timeout: 120
-  #      - store_artifacts:
-  #          path: testArtifacts
-  #          when: on_fail
+  testE2e:
+    executor: web
+    environment:
+      - BINARY_PATH: "/tmp/voyager/linux_amd64/gaiacli"
+      - NODE_BINARY_PATH: "/tmp/voyager/linux_amd64/gaiad"
+    steps:
+      - checkout
+      - attach_workspace:
+          at: *WORKSPACE
+      - yarn-install
+      - run:
+          name: Test
+          command: yarn serve & yarn run test:e2e
```

Comment:
Do we need to use a server for static files?

Reply:
I don't know exactly. will need to recheck.

Reply:
but I think for now let's just use this working version, you probably invested already enough time on this PR right? we'll have to touch it to include browserstack so let's move this when will be vital to do so :)

Reply:
I was trying to prepare the path to have a ~1 line change for browserstack 🤣